js/bybit.js

```diff
@@ -1493,22 +1540,53 @@ module.exports = class bybit extends Exchange {
         }
         let method = undefined;
         if (type === 'spot') {
-            method = 'publicGetSpotQuoteV1Ticker24hr';
+            method = 'publicGetSpotV3PublicQuoteTicker24hr';
         } else if (!isUsdcSettled) {
             // inverse perpetual // usdt linear // inverse futures
             method = 'publicGetV2PublicTickers';
         } else {
             throw new NotSupported (this.id + ' fetchTickers() is not supported for USDC markets');
         }
         const response = await this[method] (params);
+        //
+        // spot
+        //
+        //    {
+        //         "retCode": "0",
+        //         "retMsg": "OK",
+        //         "result": {
+        //             "list": [
+        //                 {
+        //                     "t": "1666772160002",
+        //                     "s": "XDCUSDT",
+        //                     "lp": "0.03109",
+        //                     "h": "0.03116",
+        //                     "l": "0.03001",
+        //                     "o": "0.03044",
+        //                     "bp": "0.03105",
+        //                     "ap": "0.03109",
+        //                     "v": "1362796.9",
+        //                     "qv": "41423.411932"
+        //                 },
+        //             ]
+        //         },
+        //         "retExtInfo": {},
+        //         "time": "1666772209124"
+        //     }
+        //
         const result = this.safeValue (response, 'result', []);
-        const tickers = {};
-        for (let i = 0; i < result.length; i++) {
-            const ticker = this.parseTicker (result[i]);
-            const symbol = ticker['symbol'];
-            tickers[symbol] = ticker;
+        if (this.isArray (result)) {
+            const tickers = {};
+            for (let i = 0; i < result.length; i++) {
+                const ticker = this.parseTicker (result[i]);
+                const symbol = ticker['symbol'];
+                tickers[symbol] = ticker;
+            }
+            return this.filterByArray (tickers, 'symbol', symbols);
+        } else {
+            const list = this.safeValue (result, 'list', []);
```

Comment:
you can't use `list` as an identifier name because it confuses our transpiler that's why the build is broken

Reply:
yeah, I've not yet looked at builds, because PR was not ready till now (my bad, i havent marked it draft when opened the PR).
Now, the PR is complete.